server/main.lua

```diff
@@ -50,15 +50,15 @@ exports('Revive', revivePlayer)
 ---removes all ailments, sets to full health, and fills up hunger and thirst.
 ---@param src Source
 local function heal(src)
-	lib.callback('qbx_medical:client:heal', src, false, 'full')
+	lib.callback.await('qbx_medical:client:heal', src, 'full')
```

Comment:
Same with this one, do we need to yield here for some reason?

Reply:
This should probably be awaited so that when the export returns, the player is already healed.